InventoryCatalog/Plugin/CatalogInventory/Model/ResourceModel/StockStatusFilterPlugin.php

```diff
@@ -37,11 +41,17 @@ class StockStatusFilterPlugin
      */
     private $stockStatusFilter;
 
+    /**
+     * @var StockStatusApplierInterface
+     */
+    private $stockStatusApplier;
+
     /**
      * @param StoreManagerInterface $storeManager
      * @param StockResolverInterface $stockResolver
      * @param DefaultStockProviderInterface $defaultStockProvider
      * @param StockStatusFilter $stockStatusFilter
+     * @param StockStatusApplierInterface|null $stockStatusApplier
      */
     public function __construct(
         StoreManagerInterface $storeManager,
```

Comment:
you are missing the changes made at row 60-61, see https://github.com/magento-commerce/inventory/pull/191/files#diff-482dd93a38823926cdd94ef803f61bcacce588c775d86a292f2f8a070d464116R60